presto-docs/src/main/sphinx/functions/map.rst

```diff
@@ -136,7 +136,7 @@ Map Functions
 
 .. function:: map_top_n(x(K,V), n) -> map(K, V)
 
-    Truncates map items. Keeps only the top N elements by value.
+    Truncates map items. Keeps only the top N elements by value.  Keys are used to break ties with the max key being chosen. Both keys and values should be orderable.
     ``n`` must be a non-negative integer.::
```

Comment:
```suggestion
    ``n`` must be a non-negative integer. ::
```
Nit: no space between the . and the first : results in "a non-negative integer.: " as shown in this screenshot 
<img width="774" alt="Screenshot 2024-06-18 at 10 56 21 AM" src="https://github.com/prestodb/presto/assets/7013443/e855f322-3cc4-4bb7-afba-09943ca4abb1">